app/javascript/mastodon/reducers/followed_tags.js

```diff
@@ -0,0 +1,44 @@
+import {
+  FOLLOWED_HASHTAGS_FETCH_REQUEST,
+  FOLLOWED_HASHTAGS_FETCH_SUCCESS,
+  FOLLOWED_HASHTAGS_FETCH_FAIL,
+  FOLLOWED_HASHTAGS_EXPAND_REQUEST,
+  FOLLOWED_HASHTAGS_EXPAND_SUCCESS,
+  FOLLOWED_HASHTAGS_EXPAND_FAIL,
+} from 'mastodon/actions/tags';
+import { Map as ImmutableMap, List as ImmutableList, fromJS } from 'immutable';
+
+const initialState = ImmutableMap({
+  followed_tags: ImmutableMap({
+    items: ImmutableList(),
+    isLoading: false,
+    next: null,
+  }),
```

Comment:
This is incorrectly nested:
```suggestion
  items: ImmutableList(),
  isLoading: false,
  next: null,
```